round.py

```diff
@@ -200,9 +224,48 @@ async def challenge(self, ctx, *users: discord.Member):
                 await ctx.send(f"{i.name} is already in a round!")
                 return
 
+        check = False 
+        start_time = get_time()
+        First = True
+        while get_time() - start_time < 60:
+            if First:
+                alts = await self.get_alt_response(self.client, ctx, f"{ctx.author.mention} add alts of users\ntype None if not applicable \nFormat \"Alts: handle_1 handle_2 .. \n\"len(handles) <= 2*len(users) must be satisfied", 30, ctx.author)
+                First = False
+            else:
+                alts = await self.get_alt_response(self.client, ctx, f"{ctx.author.mention} add alts of users", 30, ctx.author)
+            if alts[0]:
+                alts = alts[1]
+                if len(alts) < 1:
+                    continue
+                if len(alts) == 1:
+                    if alts[0] == "None":
+                        check = True
+                        alts = []
+                        break
+                else:
+                    alts = alts[1:]
+                    check = True
+                    for alt in alts:
+                        res = await self.api.check_handle(alt)
+                        if not res[0]:
+                            await ctx.send(f"{ctx.author.mention} " + alt + " is not valid codeforces handle, try again")
+                            check = False 
+                            break
+                    if check:
+                        break
+        if not check: 
+            await ctx.send(f"{ctx.author.mention} you took too long to decide")
+            return 
+
         await ctx.send(embed=discord.Embed(description="Starting the round...", color=discord.Color.green()))
 
         handles = [self.db.get_handle(ctx.guild.id, user.id) for user in users]
+        handles.extend(alts)
+        handles = list(set(handles))
+
+        if len(handles) > 2*len(users) and ctx.guild.id != 598608730464190486:
+            await ctx.send(f"{ctx.author.mention} len(handles) <= 2*len(users) must be satisfied")
+            return
```

Comment:
This should be checked inside the while loop and not outside. Also remove the guild specific feature

Reply:
ok will check inside the loop
but please let the guild specific feature stay